src/puppetlabs/puppetdb/command.clj

```diff
@@ -456,28 +458,30 @@
 (defn exec-command
   "Takes a command object and processes it to completion. Dispatch is
    based on the command's name and version information"
-  [{:keys [command version] :as cmd} db conn-status start]
+  ([{:keys [command version] :as cmd} db conn-status start]
+    (exec-command cmd db conn-status start {}))
```

Comment:
`exec-store-report` needs the `options-config` every time so we should require it as an argument